queue-health/graph.py

```diff
@@ -99,6 +99,7 @@
         plot.axvspan(start, end, alpha=0.3, color='red', linewidth=0)
 
 
+last_week = datetime.datetime.now() - datetime.timedelta(days=7)
 week_happiness = [0, 0]
```

Comment:
I feel like this is pretty icky. If you want two variables use two variables. I guess that's not this PR, though.